cmd/tsbs_generate_queries/main.go

```diff
@@ -51,6 +53,24 @@ var useCaseMatrix = map[string]map[string]utils.QueryFillerMaker{
 		iot.LabelDailyActivity:                 iot.NewDailyTruckActivity,
 		iot.LabelBreakdownFrequency:            iot.NewTruckBreakdownFrequency,
 	},
+	"finance": {
+		finance.LabelLastPrice:                     finance.NewLastPrice,
+		finance.LabelOpenHighLowClose + "-5m":      finance.NewOpenHighLowClose(5 * time.Minute),
+		finance.LabelOpenHighLowClose + "-1h":      finance.NewOpenHighLowClose(1 * time.Hour),
+		finance.LabelOpenHighLowClose + "-24h":     finance.NewOpenHighLowClose(24 * time.Hour),
+		finance.LabelMovingAverageRSI + "-24h-5m":  finance.NewMovingAverageRSI(24*time.Hour, 5*time.Minute),
+		finance.LabelMovingAverageRSI + "-24h-1h":  finance.NewMovingAverageRSI(24*time.Hour, 1*time.Hour),
+		finance.LabelMovingAverageRSI + "-24h-24h": finance.NewMovingAverageRSI(24*time.Hour, 24*time.Hour),
+		finance.LabelMovingAverageRSI + "-7d-1h":   finance.NewMovingAverageRSI(7*24*time.Hour, 1*time.Hour),
+		finance.LabelMovingAverageRSI + "-7dh-24h": finance.NewMovingAverageRSI(7*24*time.Hour, 24*time.Hour),
+		finance.LabelMovingAverageRSI + "-30d-24h": finance.NewMovingAverageRSI(30*24*time.Hour, 24*time.Hour),
+		finance.LabelTopPercentChange + "-24h-5m":  finance.NewTopPercentChange(24*time.Hour, 5*time.Minute),
```

Comment:
Also curious how the data is generated? Will we have enough data to cover the span?